Makefile

```diff
@@ -1,27 +1,30 @@
-.PHONY: module.tar.gz ur5e-sim format test run-clang-tidy run-clang-check clean clean-all docker docker-build docker-amd64 docker-upload appimages docker-arm64-ci docker-amd64-ci
-default: build/universal-robots
+.PHONY: module.tar.gz ur5e-sim format format-check test run-clang-tidy run-clang-check clean clean-all docker docker-build docker-amd64 docker-upload appimages docker-arm64-ci docker-amd64-ci
+default: build
 
 # format the source code
 format:
 	ls src/viam/ur/module/*.*pp | xargs clang-format-19 -i --style=file
 
-build:
```

Comment:
the cmake config generation will instantiate the `build` folder already.
confirmed this test passing in current PR.